tests/e2e/support.js

```diff
@@ -1,24 +1,45 @@
 import '@agoric/synpress/support/index';
-import { FACUET_HEADERS, phrasesList, DEFAULT_TIMEOUT } from './utils';
-
-const networkPhrases = phrasesList[Cypress.env('AGORIC_NET')];
+import { FACUET_HEADERS } from './utils';
 
 Cypress.Commands.add(
   'provisionFromFaucet',
   (walletAddress, command, clientType) => {
+    const TRANSACTION_STATUS = {
+      FAILED: 1000,
+      NOT_FOUND: 1001,
+      SUCCESSFUL: 1002,
+    };
+
+    const getStatus = txHash =>
+      cy
+        .request({
+          method: 'GET',
+          url: `https://${Cypress.env(
+            'AGORIC_NET'
+          )}.faucet.agoric.net/api/transaction-status/${txHash}`,
+        })
+        .then(resp => {
+          const { transactionStatus } = resp.body;
+          if (transactionStatus === TRANSACTION_STATUS.NOT_FOUND)
+            // eslint-disable-next-line cypress/no-unnecessary-waiting
+            return cy.wait(2000).then(() => getStatus(txHash));
```

Comment:
Do we have in general timeout limit on our tests? Thinking of a case when we are getting consistent `NOT_FOUND` for some unexpected reason.

Reply:
Yes there is a maximum timeout limit of 30 seconds